PublishedModules/mjlill/Astronauts/Astronauts.py

```diff
@@ -42,12 +42,15 @@ def onMessage(self, intent: str, session: DialogSession) -> bool:
 				if not amount:
 					text = managers.TalkManager.getrandomTalk('noAstronauts')
 				elif amount == 1:
-					astronauts = data['people'][0]['name']
-					text = managers.TalkManager.randomTalk(module=self.name, talk='oneAstronaut').format(astronauts)
+					text = managers.TalkManager.randomTalk(module=self.name, talk='oneAstronaut').format(
```

Comment:
It's correct but if you want to be fast you can use the helper self.randomTalk(talk)